index.d.ts

```diff
@@ -0,0 +1,35 @@
+import fastify = require("fastify");
```

Comment:
I think those come witha license header that we should likely maintain here with some notes.

Reply:
I modified the typings (last commit) to use fastify.Plugin and did a minor change on the setHeaders function. Since it is not direct copy of the typings from definitelyTyped I should we still include the header?

Reply:
The creator is a colleague of mine, I'll consult with him.

Reply:
We could follow the DefinitelyTyped proposal: https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package

```
// Definitions by: Jannik <https://github.com/jannikkeye>
//                 Leo <https://github.com/leomelzer>
```

Reply:
That sounds good. I'll set it like you proposed and omit all the DefinitelyTyped related stuff.